lib/core/random_image.dart

```diff
@@ -0,0 +1,14 @@
+// ignore_for_file: unused_import
+
+import 'package:flutter/material.dart';
+
+class RandomImage extends StatelessWidget {
+  const RandomImage({super.key, this.height = 100});
```

Comment:
And in general, give a space between constructor and class variables to improve readability.

```suggestion
  const RandomImage({super.key, this.height = 100});
  
  final imageUrl = "https://picsum.photos/200/300";
```

Reply:
Done.